packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats-modal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { ActionSheet } from '@deriv-com/quill-ui';
+import { Text } from '@deriv-com/ui';
+import StatsRow from './accumulator-stats-row';
+import { Localize } from '@deriv/translations';
+
+const AccumulatorStatsModal = ({
+    rows,
+    animationClass,
+    isMovingTransition,
+}: {
+    rows: number[][];
+    animationClass?: string;
+    isMovingTransition: boolean;
+}) => {
+    return (
+        <ActionSheet.Portal shouldCloseOnDrag>
+            <div className='stats-sheet'>
+                <div className='stats-sheet__title'>
+                    <Text size='lg' bold>
```

Comment:
Btw later we can do a small refactoring if you want: `as='div'` can be added to Quill component and `className` from div can me moved to `<Text/>`.